client/knowledge_base_search/src/views/NoResults/NoResults.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './styles.css';
+
+export default function NoResults() {
+  return (
+    <div className="_container-center">
+      <h2>No Results</h2>
+    </div>
+  );
+}
```

Comment:
Looks like you might just be getting started on creating a `views` directory. No need for this now, but you might want to consider how you'll expand on organizing this directory as it grows. It didn't take too long before I realized I had standard element views mixed in with somewhat specific views like this one and `ShowMoreResults`.

Reply:
good point - wasn't sure where to put shared components